program/Makefile.am

```diff
@@ -9,6 +9,7 @@ pdfgrader_LDADD = $(pdfgrader_LIBS)
 pdfgrader_VALAFLAGS = --pkg=gtk+-3.0 --pkg=poppler-glib --pkg cairo --pkg gee-0.8 --pkg glib-2.0 --pkg libxml-2.0 --pkg posix
 
 pdfgrader_SOURCES = src/main/System.vala \
+    src/main/ExamImage.vala \
```

Comment:
Different indent

Reply:
For some reason this didn't show up on my local code editor. Fixed it directly from github - thanks for pointing this out!